my_list.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 however your `MyList` class should implements `#each` method that yields all members of `@list` in succession.

   ```ruby
   class MyList
      include MyEnumerable

      def initialize(*list)
         @list = *list
      end

      def each(&block)
         @list.each(&block)
      end
   end
   ```
- You can check these two [repo1](https://github.com/ichala/Ruby-Enumerable/pull/1/) and [repo2](https://github.com/rbanos-mv/enumerable/pull/1/) if you have any problems 👍🏻

Reply:
Hi @BenMukebo, Thanks for the feedback.
We have resolved the issue. 🙏🏾